src/index.js

```diff
@@ -34,15 +37,10 @@ export const NightRegion = (props) => {
     return latLng
   }
 
-  useEffect(() => {
+  useInterval(async () => {
```

Comment:
Does the arrow function need to be async?

Reply:
No doesn't look like it. Not sure how that got in but fixing now.